src/lib/layer-manager.js

```diff
@@ -42,7 +42,7 @@ export default class LayerManager {
     this.layers = [];
     // Tracks if any layers were drawn last update
     // Needed to ensure that screen is cleared when no layers are shown
-    this.drewLayers = false;
+    this.layerCleared = false;
```

Comment:
Naming? This is not tracked per layer, right. Should it be called `screenCleared`?